google/cloud/bigtable/rpc_retry_policy.cc

```diff
@@ -48,38 +44,24 @@ void LimitedErrorCountRetryPolicy::setup(
     grpc::ClientContext& /*unused*/) const {}
 
 bool LimitedErrorCountRetryPolicy::on_failure(grpc::Status const& status) {
```

Comment:
For future: `on_failure` seems like it should be `OnFailure()` since it's not a trivial setter/getter function, and the forwarding method is correctly named.

Reply:
Yeah, one of those things I hate in the Google Style Guide: the name leaks details about the implementation.  Created issue and TODO entry.